Scenarios/CustomizingALGoForGitHub.md

```diff
@@ -0,0 +1,282 @@
+| Recurse | Include all files matching the file name pattern in FileSpec from all subfolders (under a given subfolder from FileSpec) | No | true |
+| AuthTokenSecret | The name of a secret (like ghTokenWorkflow) which contains a Personal Access Token or GitHub App specification with access to the files. |
+
+This setting will cause AL-Go for GitHub to include these files during the next update.
+
+> [!WARNING]
+> You can override existing AL-Go for GitHub system files this way, please prefix files in your repository with `my` or your organization name (except for DeployTo and DeliverTo) in order to avoid overriding future workflows from AL-Go for GitHub.
+
+> [!NOTE]
+> If the destination is in the .AL-Go folder, the file(s) will be copied to all .AL-Go folders in multi-project repositories.
+
+> [!NOTE]
+> If the source is a private or internal GitHub repository and you are using a GitHub app for authentication, you should use https://api.github.com/repos/OWNER/REPO/zipball/BRANCH instead of https://github.com/OWNER/REPO/archive/refs/heads/BRANCH.zip.
+
+### Adding custom jobs
+
+You can also add custom jobs to any of the existing AL-Go for GitHub workflows. Custom jobs can depend on other jobs and other jobs can made to depend on custom jobs. Custom jobs needs to be named `CustomJob<something>`, but can specify another name to be shown in the UI. Example:
+
+```yaml
+  CustomJob-CreateBuildTag:
+    name: Create Build Tag
+    needs: [ Initialization, Build ]
+    if: (!cancelled()) && (needs.Build.result == 'success')
+    runs-on: [ ubuntu-latest ]
+    steps:
+      - name: Create Tag
+        run: |
+          Write-Host "Create Tag"
+
+  PostProcess:
+    needs: [ Initialization, Build2, Build1, Build, Deploy, Deliver, DeployALDoc, CustomJob-CreateBuildTag ]
+    if: (!cancelled())
+    runs-on: [ windows-latest ]
+    steps:
+      ...
+```
+
+Adding a custom job like this, will cause this job to run simultaneously with the deploy and the deliver jobs.
+
+> [!NOTE]
+> All custom jobs will be moved to the tail of the yaml file when running Update AL-Go System Files, but dependencies to/from the custom jobs will be maintained.
+
+> [!CAUTION]
+> Custom jobs might be broken if the customized AL-Go for GitHub workflow has been refactored and the referenced jobs have been renamed.
+
+### Modifying workflow permissions
+
+If any of your custom jobs require permissions, which exceeds the permissions already assigned in the workflow, then these permissions can be specified directly on the custom job.
+
+If any of your custom steps require permissions, which exceeds the permissions already assigned in the workflow, you can modify the permissions of the workflow and assign additional permissions. AL-Go for GitHub will not allow you to remove permissions, which might be needed in other steps/jobs, but additional permissions will be included when running Update AL-Go System Files.
```

Comment:
Is this supported? In my repo it's just removed again 🤷‍♂️ 
![image](https://github.com/user-attachments/assets/ac689ab8-0106-4d8c-b64f-598a8b941835)